app/src/lib/release-notes.ts

```diff
@@ -106,10 +109,23 @@ export async function getChangeLog(
   }
 }
 
-export async function generateReleaseSummary(): Promise<ReleaseSummary> {
-  const releases = await getChangeLog()
-  const latestRelease = releases[0]
-  return getReleaseSummary(latestRelease)
+export async function generateReleaseSummary(): Promise<
+  ReadonlyArray<ReleaseSummary>
+> {
+  const lastTenReleases = await getChangeLog()
+  const currentVersion = new semver.SemVer(getVersion())
+  const recentReleases = lastTenReleases.filter(
+    r =>
+      semver.gte(new semver.SemVer(r.version), currentVersion) &&
+      new Date(r.pub_date).getTime() > offsetFromNow(-90, 'days')
+  )
+
+  // We should only be pulling release notes when a release just happened, so
+  // there should be one within the past 90 days. Thus, this is just precaution
+  // to ensure we always show at least the last set of release notes.
+  return recentReleases.length > 0
+    ? recentReleases.map(getReleaseSummary)
+    : [getReleaseSummary(recentReleases[0])]
```

Comment:
Shouldn't this be `lastTenReleases`?
```suggestion
    : [getReleaseSummary(lastTenReleases[0])]
```

Reply:
Yep.. I renamed variables and overlooked here. Thanks!